name: '404'
channels:
  - defaults
dependencies:
  - ca-certificates=2021.4.13=haa95532_1
  - certifi=2020.12.5=py38haa95532_0
  - openssl=1.1.1k=h2bbff1b_0
  - pip=21.0.1=py38haa95532_0
  - python=3.8.5=h5fd99cc_1
  - setuptools=52.0.0=py38haa95532_0
  - sqlite=3.35.4=h2bbff1b_0
  - vc=14.2=h21ff451_1
  - vs2015_runtime=14.27.29016=h5e58377_2
  - wheel=0.36.2=pyhd3eb1b0_0
  - wincertstore=0.2=py38_0
  - pip:
    - asgiref==3.3.1
    - distlib==0.3.1
    - dj-database-url==0.5.0
    - django==3.1.7
    - django-cors-headers==3.7.0
    - django-heroku==0.3.1
    - djangorestframework==3.12.3
    - psycopg2==2.8.6
    - pypiwin32==223
    - scipy==1.6.2
    - sqlparse==0.4.1
    - virtualenv==20.4.3
    - whitenoise==5.2.0
prefix: E:\ANACONDA\envs\404
